modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the images provided in the client repository for this operating system are out of date.
+Bootstrapping new {centos} 6 clients using these packages will fail.
+This failure will not happen on {centos} 6 clients that are already installed and bootstrapped.
+
+If you need to bootstrap new {centos} 6 clients you can edit the existing repositories to reflect the correct URL.
+
+
+
+.Procedure: Troubleshooting Bootstrapping a New {centos} 6 Client
+. On the {centos} 6 client, at the command prompt, open the ``CentOS-Base.repo`` file, located in the `/etc/yum/repos.d/` directory.
+. Locate the ``baseurl`` line, pointing to ``mirrorlist.centos.org``.
+There may be more than one entry.
+For example:
++
+----
+baseurl=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
```

Comment:
Err... `release=7`? This is about CentOS6, isn't it?

Reply:
yes, sorry. As 6 went already away, I had to copy the centos 7 entry as example.
@Loquacity just change `7` to `6` and we are good here.